packages/components/psammead-topic-tags/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Topic Tags Changelog
+
+| Version | Description |
+|---------|-------------|
+| 1.0.0   | [PR#4248](https://github.com/BBC-News/psammead/pull/4248) Initial creation of package. |
```

Comment:
Should have alpha version number until it's passed the accessiblity swarm?

Reply:
Thanks for the comments @greenc05,

Nice spot, on it!